src/Services/GatewayAccountService.php

```diff
@@ -96,4 +97,21 @@ public function delete($gatewayAccountId)
     {
         $this->client()->delete('gateway-accounts/{gatewayAccountId}', ['gatewayAccountId' => $gatewayAccountId]);
     }
+
+    /**
+     * @param string $gatewayAccountId
+     * @param array|JsonSerializable $data
+     *
+     * @throws NotFoundException The resource data does not exist
+     *
+     * @return bool
+     */
+    public function checkCredentials($gatewayAccountId, $data = [])
+    {
+        try {
+            return empty($this->client()->post($data, 'gateway-accounts/{gatewayAccountId}/check-credentials', ['gatewayAccountId' => $gatewayAccountId]));
+        } catch (DataValidationException $ex) {
+            return false;
```

Comment:
Not sure about suppressing `DataValidationException` though. That makes impossible to understand the errors if happen.